mitodl/open-discussions
# Import Sloan Executive courses

#### Pre-Flight checklist


#### What are the relevant tickets?
- Closes #2712 

#### What's this PR do?
- Imports courses from Sloan (https://executive.mit.edu/)

#### How should this be manually tested?
- set `SEE_BASE_URL=https://executive.mit.edu/` in your .env file
- run `python manage.py backpopulate_see_data`

It should complete without errors, and there should be 69 new courses that should show up in search offered by 'Sloan Executive Education'

The reason I made `SEE_BASE_URL` a setting is so we can turn off imports if we ever need/want to by leaving it blank/null.

#### Screenshots (if appropriate)
<img width="1249" alt="Screen Shot 2020-04-01 at 11 04 50 AM" src="https://user-images.githubusercontent.com/187676/78156557-f5c55000-740c-11ea-892b-7e98054e0e48.png">

<img width="492" alt="Screen Shot 2020-04-02 at 9 16 00 AM" src="https://user-images.githubusercontent.com/187676/78276736-73f32680-74e1-11ea-9840-77310ed6de3e.png">
